tailwind.js

```diff
@@ -213,6 +213,8 @@ module.exports = {
         '3xl': '1.875rem', // 30px
         '4xl': '2.25rem', // 36px
         '5xl': '3rem', // 48px
+        '7xl': '4rem', // 64px
```

Comment:
Can you confirm all these new additions are actually being used? I don't see any references to them in the file `resources/views/components/video.blade.php`. Maybe I'm missing something though!

Reply:
I will confirm and double check thanks!!